bool fan
bool fanError
int8 wettingAgigator
bool wettingAgigatorError
bool waterRelay
int8 waterFlowRate

